### DISCUSSION TITLE
Any way to show placeholder without datasource?

### DISCUSSION THREAD

**Discussion by sleewok** [jhomlala/betterplayer#540] [Q&A]
Is there any way to load BetterPlayer to show the placeholder and when clicked have it load the datasource? Or at least not make the network connection until clicked? 

I don't want to establish a connection unless the user actually decides to view the video.

I know that I can setup a listener and display an image and when I click it have it load BetterPlayer, but I'm hoping there is a way to do this directly with BetterPlayer.

Thanks!

**Comment by jhomlala** [comment]
There's no other way than you've already suggested. Just create image widget and then once user clicks show video.
